ebean-migration/src/test/java/io/ebean/migration/MigrationTableAsyncTest.java

```diff
@@ -151,6 +170,21 @@ private void runTest() throws SQLException, InterruptedException, ExecutionExcep
     dropTable("m1");
     dropTable("m2");
     dropTable("m3");
+    
+    if (withExisting) {
+      // create empty migration table
+      try (Connection conn = dataSource.getConnection()) {
+        String derivedPlatformName = DbNameUtil.normalise(conn);
+
+        config.setPlatform(derivedPlatformName);
```

Comment:
I forgot this line, and got wrong script for sql server (see other comment with exception handling)